docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge
```

Comment:
It is not critical but in general practice, networks block writes after services